### DISCUSSION TITLE
parameter for fullscreen and choosing monitor?

### DISCUSSION THREAD

**Discussion by DannyDarth** [streamlink/streamlink-twitch-gui#989] [Q&A] [answered]
I know the parameter for fullescreen is --fs. But is there something i can put behind it to choose the monitor i want it to open on?

**Comment by bastimeyer** [comment, marked as answer]
> `--fs`

What are you talking about? You must be talking about player parameters... You can set as many arbitrary player parameters as you want. This has nothing to do with Streamlink's or Streamlink Twitch GUI's parameters.
https://github.com/streamlink/streamlink-twitch-gui/wiki/Player-configuration#custom-player-parameters

Check your player's documentation.

**Reply by bastimeyer** [reply]
https://mpv.io/manual/stable/
https://mpv.io/manual/stable/#window

**Reply by DannyDarth** [reply]
What i mean is the window opens on my main monitor and i want it to open on my 3rd monitor.

**Reply by mkbloke** [reply]
Yes, we know what you mean.  You will need to consult the documentation for your chosen video player to do that.  As stated above, this is not a Streamlink issue.

**Reply by bastimeyer** [reply]
> the window

This is the Streamlink Twitch GUI discussion forum. If you ask about "the window", then I have to interpret this as the Streamlink Twitch GUI window. But you're clearly talking about your player, whose name you also didn't even mention. From the context though, it's MPV.

I won't provide any help about any players here. I have also already linked you the MPV docs, even the right section with the parameters you need. Set the required MPV launch parameters in the player parameters fields in the Streamlink Twitch GUI settings. Good luck.
